modules/imaging_browser/php/NDB_Form_imaging_browser.class.inc

```diff
@@ -186,18 +186,32 @@ class NDB_Form_Imaging_Browser extends NDB_Form
                 }
             }
 
+            // Provide a default pic for DTI with time dimention / volumes
+            // processed before 2012-07, after this date, they are fixed.
+            $config = NDB_Config::singleton();
+            $check_pic_filename = $FileObj->getParameter('check_pic_filename');
+            $Time = number_format($FileObj->getParameter('time'), 2);
+            if (!file_exists(
+                $config->getSetting('imagePath') . "pic/" .
+                $check_pic_filename
+            ) && $Time > 0) {
+                $check_pic_filename = str_replace(
+                    '_check.jpg',
+                    '_check-0.jpg',
```

Comment:
Hi @gluneau @MounaSafiHarab ,

I feel this is an issue specific to IBIS and therefore it probably should not be imported into the main LORIS. 
I checked the back end of PREVENT-AD, we never had any _check-0.jpg image although we have a lot of 4D datasets (DWI, BOLD and ASL). The pic shows up nicely since the beginning of the project too.

@MounaSafiHarab PREVENT-AD started acquiring data in July 2012. Are there any other projects than IBIS that started before 2012?